src/ProcessSlave.php

```diff
@@ -195,15 +197,19 @@ function ($item) {
         }
 
         if (!$this->loop) {
```

Comment:
Change to if this loop -> stop to simplify?

Reply:
Good point, I simplified it